### DISCUSSION TITLE
need to increase mermaid version

### DISCUSSION THREAD

**Discussion by stumash** [community/community#14868] [Other Feature Feedback, Questions, & Ideas]
Mermaid recently released a new version (8.14) where a bunch of bugs got fixed - most notably in the rendering of `erDiagram`s.

`erDiagram`s that I put in `README.md` files still have rendering issues that were solved in the new version. For example,
the `pk`/`fk` column renders on top of the `name` column.

please increase the mermaid version you use for `.md` rendering so that we can all benefit from bugfixes to mermaid

**Comment by injecteer** [comment]
As of now, the latest version of mermaid `10.4.0` should be installed, as it fixes several issues.
When can we count on that upgrade?
